internal/resources/resource_network.go

```diff
@@ -158,6 +158,11 @@ func networkSchema() map[string]*schema.Schema {
 			Default:     rest.NETWORKHOSTUSE_OPTIONAL,
 			Description: "Required, Optional or Default",
 		},
+		nPurpose: {
+			Type:        schema.TypeString,
+			Optional:    true,
+			Description: getSupportedNetworkPurpose(),
+		},
```

Comment:
how about including field descirption + allowed values as part of the description ?
"Purpose of the network. Allowded values are x, y, z"